code/_onclick/ai.dm

```diff
@@ -166,15 +166,15 @@
 /obj/machinery/power/apc/AICtrlClick() // turns off/on APCs.
 	if(can_use(usr, 1))
 		toggle_breaker()
-		add_fingerprint(usr)
+		add_fingerprint_from_mob(usr)
```

Comment:
as I said a while back in irc, revert this change and it'll be far less annoying to review this